lab_1/main.py

```diff
@@ -11,7 +12,19 @@ def tokenize(text: str) -> list or None:
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
-    pass
+    if not isinstance(text, str):
+        return None
+
+    low_text = text.lower()
+    clean_text = ''
+    for i in low_text:
+        if i.isalpha():
```

Comment:
add the next check here (`i.isspace()`), no need for the duplicate code `+=i`

Reply:
done